src/Pages/Cart/CartList/CartItem/CartItem.js

```diff
@@ -0,0 +1,202 @@
+      });
+
+    category === 'vitamins' &&
+      this.setState({
+        vitaminQuantity: productQuantity,
+      });
+  }
+
+  componentDidUpdate() {
+    const { target, productId } = this.state;
+
+    target === 'powders' && this.updatePowderCart(productId);
+    target === 'vitamins' && this.updateVitaminCart(productId);
+  }
+
+  updatePowderCart = productId => {
+    console.log('파우더 변경');
+
+    fetch(`${SERVER}/order/cart/${productId}`, {
+      method: 'POST',
+      headers: {
+        Authorization: sessionStorage.getItem('access_token'),
+      },
+      body: JSON.stringify({
+        productId: this.state.productId,
+        productSize: this.state.powderSize,
+        productQuantity: this.state.powderQuantity - 0,
+      }),
+    })
+      .then(response => response.json())
+      .then(res => console.log(res));
+
+    console.log(
+      this.state.productId,
+      this.state.powderQuantity - 0,
+      this.state.powderSize,
+    );
+  };
+
+  updateVitaminCart = productId => {
+    console.log('비타민 변경');
+
+    fetch(`${SERVER}/order/cart/${productId}`, {
+      method: 'POST',
+      headers: {
+        Authorization: sessionStorage.getItem('access_token'),
+      },
+      body: JSON.stringify({
+        productId: this.state.productId,
+        productQuantity: this.state.vitaminQuantity - 0,
+      }),
+    })
+      .then(response => response.json())
+      .then(res => console.log(res));
+
+    console.log(this.state.productId, this.state.vitaminQuantity);
+  };
+
+  handleVitaminOption = (itemInfo, event) => {
+    this.setState({
+      target: itemInfo.category,
+      productId: itemInfo.productId,
+      vitaminQuantity: event.target.value,
+    });
+  };
+
+  handlePowderOption = (itemInfo, event) => {
+    const { name, value } = event.target;
+
+    name === 'productSize' &&
+      this.setState({
+        powderSize: value,
+        target: itemInfo.category,
+        productId: itemInfo.productId,
+      });
+
+    name === 'productQuantity' &&
+      this.setState({
+        powderQuantity: value,
+        target: itemInfo.category,
+        productId: itemInfo.productId,
+      });
+  };
+
+  render() {
+    const { itemInfo, pruducts, deleteItem } = this.props;
+
+    return (
+      <div
+        className={`cartItem item ${pruducts.length > 3 && 'grid'}`}
+        key={itemInfo.productStockId}
+      >
+        <img src={itemInfo.productImageUrl} alt="product image" />
+        <div>
+          <div>
+            <h2>{itemInfo.productName}</h2>
+            <p>{itemInfo.productSubName}</p>
+          </div>
+
+          <p className="price">${itemInfo.productPrice}</p>
+          {itemInfo.category === 'vitamins' && (
+            <div className="selectOption">
+              <select
+                onChange={() => this.handleVitaminOption(itemInfo, event)}
+              >
+                <option
+                  selected={itemInfo.productQuantity === 1 ? true : false}
```

Comment:
불리안 값을 이용하면 true, false를 굳이 더 작성하지 않으셔도 됩니다.
```suggestion
                  selected={itemInfo.productQuantity === 1}
```